cli/azd/pkg/infra/provisioning/bicep/prompt.go

```diff
@@ -122,14 +133,6 @@ func (a *BicepProvider) locationsWithQuotaFor(
 	var iterationError error
 	sharedResults.Range(func(location, quotaDetails any) bool {
 		usages := quotaDetails.([]*armcognitiveservices.Usage)
-		hasS0SkuQuota := slices.ContainsFunc(usages, func(q *armcognitiveservices.Usage) bool {
-			// The minimum quota for the S0 SKU in Microsoft.CognitiveServices/accounts is 2 capacity units
-			return *q.Name.Value == "OpenAI.S0.AccountCount" && (*q.Limit-*q.CurrentValue) >= 2
-		})
-		if !hasS0SkuQuota {
-			// If the S0 SKU quota is not available, skip this location
-			return true
-		}
```

Comment:
How come we're removing this quota check?

Reply:
I thought it was not required any longer... but on second thought, we still need it